src/modules/todo.js

```diff
@@ -0,0 +1,139 @@
+import deleteIcon from '../assets/icons/delete.png';
+import editIcon from '../assets/icons/edit.png';
+
+const iconRemove = new Image();
+iconRemove.src = deleteIcon;
+
+const iconEdit = new Image();
+iconEdit.src = editIcon;
+
+const todoList = [];
+
+export default class ToDo {
+  constructor(todo) {
+    this.description = todo;
+    this.index = todoList.length + 1;
+    this.completed = false;
+    this.checked = false;
+  }
+
+  static addTodo() {
+    const todoform = document.getElementById('todo-form');
+    const todoinput = document.getElementById('todo-input');
+    let todoList = [];
+    todoform.addEventListener('submit', (e) => {
+      e.preventDefault();
+      todoList = JSON.parse(localStorage.getItem('todoList') || '[]');
```

Comment:
You can make a function for this code line such as getdata() and call it when required.